src/shabdawali.js

```diff
@@ -1,309 +1,334 @@
-var shuffle = require('./util').shuffle;
-var replaceOn = require('./util').replaceOn;
-var commonStartingString = require('./util').commonStartingString;
-
-function Shabdawali(targetEl, opts){
-    this.element = targetEl;
-    if(!opts) opts={};
-
-    this.lines = opts.lines;
-    this.playCount = -1;
-    this.onChar = opts.onCharChange || function(){};
-    this.onLine = opts.onLineChange || function(){};
-    this.nextWord = opts.nextWord || function(){};
-
-    this.speed = opts.typingSpeed || 70;
-    this.timeToReadAWord = 80;
-    this.dynamicPauseBeforeDelete = opts.dynamicPauseBeforeDelete || true; 
-    this.pauseBeforeDelete = opts.pauseBeforeDelete || 2000; 
-    this.pauseBeforeNext = opts.pauseBeforeNext || 1000; 
-    this.delay = opts.delay || 0; //initial delay
-
-    this.typoEffect = opts.typoEffect || false;
-
-    this.deleteSpeed = opts.deleteSpeed || (this.speed / 2);
-    this.deleteSpeedArr = [];
-    this.deleteUpto = [];
-
-    if(opts.repeat === false){
-        this.repeat = false;
-    }else{
-        this.repeat = true;
-    }
-
-    if(opts.deleteEffect === false){
-        this.deleteEffect = false;
-    }else{
-        this.deleteEffect = true;
-    }
-
-    this.appendCursor(this.element);
-
-    opts.cursorEffect ? this.showCursor() : this.hideCursor();
-
-    if(opts.deleteFrom === "start"){
-        this.trimmedText = function(text,len){ return text.substring(1); }
-    }else{
-        this.trimmedText = function(text,len){ return text.substring(0, len); }
-    }
+(function(f){if(typeof exports==="object"&&typeof module!=="undefined"){module.exports=f()}else if(typeof define==="function"&&define.amd){define([],f)}else{var g;if(typeof window!=="undefined"){g=window}else if(typeof global!=="undefined"){g=global}else if(typeof self!=="undefined"){g=self}else{g=this}g.shabdawali = f()}})(function(){var define,module,exports;return (function(){function r(e,n,t){function o(i,f){if(!n[i]){if(!e[i]){var c="function"==typeof require&&require;if(!f&&c)return c(i,!0);if(u)return u(i,!0);var a=new Error("Cannot find module '"+i+"'");throw a.code="MODULE_NOT_FOUND",a}var p=n[i]={exports:{}};e[i][0].call(p.exports,function(r){var n=e[i][1][r];return o(n||r)},p,p.exports,r,e,n,t)}return n[i].exports}for(var u="function"==typeof require&&require,i=0;i<t.length;i++)o(t[i]);return o}return r})()({1:[function(require,module,exports){
```

Comment:
Can you please explain the purpose of this change?

Reply:
It was added mistakenly from the auto-generated code. Sincere apologies. Does updating the source branch update the PR? I updated the forked repo. to remove this.